ecosystem/cli/website.py

```diff
@@ -62,7 +62,10 @@ def _load_from_file(
     )
 
     # Label descriptions
-    label_descriptions = json.loads((resources_dir / "labels.json").read_text())
+    label_descriptions = {}
+    for _category in json.loads((resources_dir / "labels.json").read_text()).values():
+        for label in _category:
+            label_descriptions[label["name"]] = label["description"]
```

Comment:
The current website doesn't use "categories", so this logic flattens the file back into the original structure.

Reply:
Sg! It'd be helpful to add a brief comment